packages/api-client/src/client/types.ts

```diff
@@ -18,22 +20,40 @@ export interface Link {
     rel: string;
 }
 
-export interface OdsDataset {
-    dataset_id?: string;
-    dataset_uid?: string;
-    has_records?: boolean;
-    data_visible?: boolean;
-    features?: string[];
+export type OdsDatasetFieldType =  typeof ODS_DATASET_FIELD_TYPE[keyof typeof ODS_DATASET_FIELD_TYPE];
```

Comment:
I added the utility type `ValueOf<>` on the platform, we can maybe add it here. (or I could make an npm package…). https://github.com/opendatasoft/platform/blob/8497ce8419cacc607c41b79ff0b87534524dddf9/ods/react/src/utils/typeScript.ts#L29

Reply:
> or I could make an npm package…
Yes, exactly, like a TypeScript helpers library. Because it would feel wrong to import in our project ts helpers from this lib

But should this library be under this repository?

Reply:
Ok so I start simply with a utils folder
https://github.com/opendatasoft/ods-dataviz-sdk/pull/259/commits/f5f1ba215f12bf31a7279995b1a7c70b5b684259